.gitignore

```diff
@@ -33,6 +33,11 @@ src/.libs/
 TAGS
 .dir-locals.el
 
+#################
+## Python
+#################
+__pycache__/
+
```

Comment:
FYI, this change has already been committed to `master` (commit 2d84c4182712432ae3402e3ccee771b5dc04e471) so it will be ignored when your PR will be merged.